fractions/javaee/batch-jberet/src/main/java/org/wildfly/swarm/batch/jberet/BatchFraction.java

```diff
@@ -73,9 +72,9 @@ public BatchFraction applyDefaults() {
 
         // Default thread-pool
         final ThreadPool<?> threadPool = new ThreadPool<>(DEFAULT_THREAD_POOL_NAME);
-        threadPool.maxThreads(ProcessorInfo.availableProcessors())
-                .keepaliveTime("time", "30")
-                .keepaliveTime("unit", "seconds");
+        threadPool.maxThreads(10)
```

Comment:
Why hard coding the number of threads?

Reply:
That's what WildFly's default config has.

Reply:
Shouldn't we offer a way to customize it, with WF you can just modify the standalone.xml but our users don't have that option

Reply:
Can be changed via the YAML, no?

Reply:
If we're setting it directly in a fraction, I'm not sure.

@bobmcwhirter might recall